app/scripts/metamask-controller.actions.test.js

```diff
@@ -157,67 +157,49 @@ describe('MetaMaskController', function () {
   });
 
   describe('#importAccountWithStrategy', function () {
-    it('two sequential calls with same strategy give same result', async function () {
```

Comment:
Importing a duplicated account will result in an error with this `KeyringController` version